teamscale-client/src/main/kotlin/com/teamscale/client/TeamscaleClient.kt

```diff
@@ -216,6 +224,10 @@ open class TeamscaleClient {
 		partition: String,
 		message: String
 	) {
+		if (projectId == null) {
+			throw IllegalArgumentException("Project ID must not be null!")
+		}
```

Comment:
This is the Java way of handling this. Kotlin actually has built in functions to check this (see https://kotlinlang.org/docs/exceptions.html)
```suggestion
		require(projectId != null) { "Project ID must not be null!" }
```